### DISCUSSION TITLE
More explanations about {{ page.translations }}

### DISCUSSION THREAD

**Discussion by happytodev** [Cecilapp/Cecil#1905] [Q&A] [answered]
Hello Arnaud,

I tried to understand by myself but... I did'nt :-(

How to use the `{{ page.translations }}` you provide in https://cecil.app/documentation/content/#link-translations-of-a-page?

You wrote "Each translated page reference the pages in others languages." but I can figure it out how. Sorry.

Can you give an example please?

Thank you.

**Comment by ArnaudLigny** [comment, marked as answer]
Hello, the goal of `page.translation` is to help to know if there is others languages of the same page, for example _download.fr.md_ is the French translation of _download.md_.

It could be useful when you want to create a language selector, for example:

- site : https://statidocs.cecil.app
- source : https://github.com/Cecilapp/theme-docs/blob/main/layouts/partials/language-selector.html.twig

**Reply by happytodev** [reply]
oh yes thanks for the resource. I think it will be nice to have more explanation in docs, because on the first read I don't understand what can I do with this variables. But maybe it's only me the problem ;-)
